aries_cloudagent/ledger/indy.py

```diff
@@ -486,95 +485,108 @@ def __init__(
 
         """
 
+        async def cred_def_in_wallet(wallet_handle, cred_def_id) -> bool:
+            """Create an offer to check whether cred def id is in wallet."""
+            try:
+                await indy.anoncreds.issuer_create_credential_offer(
+                    wallet_handle,
+                    cred_def_id
+                )
+                return True
+            except IndyError as error:
+                if error.error_code != ErrorCode.CommonInvalidStructure:
```

Comment:
Do we also need to check the content of the error message string? i.e., is it possible for `ErrorCode.CommonInvalidStructure` to be raised in other scenarios which we should allow to bubble up?

Reply:
The only other case would be a cred def id that did not parse as a cred def id, which would have been kicked out many layers above.